contrib/svc-cat-apiserver-aggregation-tls-setup.sh

```diff
@@ -63,16 +63,3 @@ echo '{"CN":"'${SVCCAT_SERVICE_NAME}'","hosts":['${ALT_NAMES}'],"key":{"algo":"r
 export SC_SERVING_CA=${SVCCAT_CA_CERT}
 export SC_SERVING_CERT=apiserver.pem
 export SC_SERVING_KEY=apiserver-key.pem
-
-cat <<EOF > $HELM_SCRIPT
```

Comment:
$HELM_SCRIPT is declared at the top, and not used anymore after you do the change.
Also another issue caused by `set -o errexit` I mentioned in the comment https://github.com/kubernetes-incubator/service-catalog/pull/1185#issuecomment-327490202
@arschles can you just revert the commit https://github.com/kubernetes-incubator/service-catalog/pull/1141/files completely to restore it to the latest stable state, so we can carefully rethink how this script should look like?

Reply:
@nilebox good catch. I will propose removing the `set -o errexit` in another PR.

And finally, I'm happy to rethink this script (possibly implement its functionality in a helm plugin?) but I want something working right now for users on as many platforms as possible.